headers/types/common/file_io.h

```diff
@@ -140,4 +140,19 @@ struct wte_header {
 };
 ASSERT_SIZE(struct wte_header, 32);
 
+struct bg_list_entry_sub_struct {
+    char name[9]; // Zero terminated string
+    undefined1 field_0x9;
```

Comment:
This is probably just a null terminator? If so; Would it maybe make sense to just include it in name?